Source/Scene/QuadtreePrimitive.js

```diff
@@ -1514,38 +1518,6 @@ function updateHeights(primitive, frameState) {
           data.callback(position);
           data.level = tile.level;
         }
-      } else if (tile.level === data.level) {
```

Comment:
This used to remove the height clamp callback from the object when the terrain tile below was refined as far as possible, possibly as an optimization since the height was never going to get more accurate or change. This code has been deleted because terrain exaggeration can happen at any time, so the callback needs to stick around.